diagnostics-app/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "diagnostics-app",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development",
+    "test": "ng test"
+  },
+  "private": true,
+  "dependencies": {
```

Comment:
Can you please add a brief comments for each dependency? It will help to understand why do we need them all.

Reply:
I've added them, please have a look.